qa/standalone/scrub/osd-scrub-test.sh

```diff
@@ -187,6 +187,48 @@ function TEST_interval_changes() {
     teardown $dir || return 1
 }
 
+function TEST_scrub_extented_sleep() {
+    local dir=$1
+    local poolname=test
+    local OSDS=3
+    local objects=15
+
+    TESTDATA="testdata.$$"
+
+    setup $dir || return 1
+    run_mon $dir a --osd_pool_default_size=3 || return 1
+    run_mgr $dir x || return 1
+    local scrub_begin_hour=$(date -d '2 hour ago' +"%H" | sed 's/^0//')
+    local scrub_end_hour=$(date -d '1 hour ago' +"%H" | sed 's/^0//')
+    for osd in $(seq 0 $(expr $OSDS - 1))
+    do
+      run_osd $dir $osd --osd_scrub_sleep=0 \
+                        --osd_scrub_extended_sleep=10 \
+                        --bluestore_cache_autotune=false \
+                        --osd_scrub_begin_hour=$scrub_begin_hour \
+                        --osd_scrub_end_hour=$scrub_end_hour || return 1
+    done
+
+    # Create a pool with a single pg
+    create_pool $poolname 1 1
+    wait_for_clean || return 1
+
+    # Trigger a scrub on a PG
+    local pgid=$(get_pg $poolname SOMETHING)
+    local last_scrub=$(get_last_scrub_stamp $pgid)
+    ceph pg scrub $pgid || return 1
```

Comment:
@Jeegn-Chen You'll need to use the trigger_scrub mechanism (see qa/standalone/scrub/osd-scrub-repair.sh) for this test to work once you check must_scrub.  The trigger_scrub fakes a scheduled scrub so that it is triggered without must_scrub being set.

Reply:
Thanks a lot for the guidance! I will work on it.